rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/widget/CovidRDTLabelFactoryTest.java

```diff
@@ -28,6 +33,17 @@
  */
 public class CovidRDTLabelFactoryTest extends BaseRDTLabelFactoryTest {
 
+    @Override
+    public void setUp() {
+        super.setUp();
+        try {
+            mockStaticMethods();
+            rdtLabelFactory = getRdtLabelFactory();
+        } catch (JSONException ex) {
+            Timber.e(ex);
+        }
```

Comment:
any reason we need this?

Reply:
On that time i was capturing the method `getOnLabelClickedListener()` in `CovidRDTLabelFactory` class from this approach. but now I have fixed the issue and removed this code snippet.